void tomo_db0::IoLoop::run_loop<false, false, false, true, true>()
--- PRE

+++ POST

@@ -571,7 +571,7 @@

 0ace [ 2] xchg %ax,%ax
 0ad0 [ 5] mov $0x1,%esi
 0ad5 [ 3] mov %rbx,%rdi
-0ad8 [ 5] call <tomo_db0::IoLoop::arm_accept(tomo_db0::UrKind)>
+0ad8 [ 5] call <tomo_db0::IoLoop::fliplisten_retry_accept(tomo_db0::UrKind)>
 0add [ 5] jmp <+0x30b>
 0ae2 [11] data16 cs nopw 0x0(%rax,%rax,1)
 0aed [ 3] nopl (%rax)
